lib/foreman_bootdisk/engine.rb

```diff
@@ -69,6 +69,7 @@ class Engine < ::Rails::Engine
         Host::Managed.include ForemanBootdisk::Orchestration::Compute if SETTINGS[:unattended]
         HostsHelper.prepend ForemanBootdisk::HostsHelperExt
         Foreman::Model::Vmware.prepend ForemanBootdisk::ComputeResources::Vmware if Foreman::Model::Vmware.available?
+        ForemanFogProxmox::Proxmox.prepend ForemanBootdisk::ComputeResources::Proxmox if ForemanFogProxmox::Proxmox.available?
```

Comment:
This will fail if `ForemanFogProxmox` is not available.  I believe you can use `defined?` here to check if `ForemanFogProxmox` is defined.

```suggestion
        ForemanFogProxmox::Proxmox.prepend ForemanBootdisk::ComputeResources::Proxmox if defined?(ForemanFogProxmox)
```
But this is untested.